src/KingSystem/Damage/dmgStruct20.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <basis/seadTypes.h>
+#include <prim/seadRuntimeTypeInfo.h>
+
+#include "KingSystem/Utils/Types.h"
+
+namespace ksys::dmg {
+
+// What exactly Struct20 is, isn't known yet.
+// It is used by the Damage Managers, especially DamageManager and up, not so much in
+// DamageManagerBase. It contains flags and values which seems to be returned and modified when
+// handling damage.
+
+// TODO: Figure out exactly what Struct20 does and rename struct + variables to fit.
+
+class Struct20Base {
+    SEAD_RTTI_BASE(Struct20Base)
+
+public:
+    virtual ~Struct20Base(){};
+
+    virtual void reset();
+    virtual void combineMaybe(Struct20Base* other);
+};
+
+class Struct20 : public Struct20Base {
+    SEAD_RTTI_OVERRIDE(Struct20, Struct20Base)
+
+public:
+    ~Struct20() { return; };
+
+    virtual void reset() override;
+
+    __attribute__((noinline)) virtual void combineMaybe(Struct20Base* other) override;
```

Comment:
```suggestion
    __attribute__((noinline)) void combineMaybe(Struct20Base* other) override;
```
(is the noinline required?)

Reply:
Yes, or else it would inline in Struct20_2::combineMaybe, which it didn't in the original.